Doc/library/sys.rst

```diff
@@ -209,6 +209,19 @@ always available.
    yourself to control bytecode file generation.
 
 
+.. data:: bytecode_path
+
+   If this is set (not ``None``), Python will write ``.pyc`` files to (and read
```

Comment:
While summarising my review comments, an alternative occurred to me that would allow `cache` to appear in the name without that name becoming excessively long: `sys.pycache_prefix` and `PYTHONPYCACHEPREFIX`.

This would also tie the config setting name directly to the fact that it is used to override the default behaviour of using `__pycache__` directories inside the source directories.